### DISCUSSION TITLE
Can't capture more then two or three videos with same quality

### DISCUSSION THREAD

**Discussion by ghost** [aclap-dev/video-downloadhelper#233] [Help and Q&A]
### Your question

Since the last update I am not able to download more than one video of the same quality from the same domain without having to reset Firefox. For example, I want to download two videos at 1080x1920 px from the same page. The first one is prefectly converted but if I want to capture a second video from the same webpage with the same resolutions, the addon acts like it is already downladed. So I must close firefox and openit again. I tried to resintall an version before the update and it keeps happening. Any ideas or help to fix this?

### Addon & CoApp version and details

version: 8.2.0.23
channel: stable
build date: 2024-02-14
build options: linuxlic: false, noyt: false, browser: firefox
lang: es-ES
license: accepted
key (only 16 first characters): a45dfb12bbf747d2
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:123.0) Gecko/20100101 Firefox/123.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.17",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\alber",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "63b936e978"
}

**Comment by GrampaWildWilly** [comment]
Give us a sample URL.  Also, instead of closing Firefox, see if reloading the extension does what you want: VDH settings -> More -> Reload extension.  Understand that performing this function will abruptly terminate any VDH downl0ads you have in progress, probably leaving you with damaged files.  You would have to download them again from the beginning.  Also, if you have any downloads queued, the queue will be discarded & you'll have to requeue whatever you had in there.

You might also try the latest VDH beta.  Paul put up a new one last night.  Click the Wiki link at the top of this page.  Then scan the Table of Contents on the right side of that page.  There is one that mentions betas.  That's where you'll find it.

**Comment by paulrouget** [comment]
Hmm… I imagine that you go to https://example.com/a, download one video, then click on a link, bringing you to https://example.com/b, and the video of that same size looks already downloaded. Correct?

Like @GrampaWildWilly said, we need a sample URL.

Also, update your addon. Current version is 8.2.1.1.

**Comment by GrzegorzDrozd** [comment]
I have the same issue. I cannot provide you a sample because it is not a public video (but it is on vimeo if that makes any difference). 

Is there a way to force download button to always be there? Or to not keep track of videos?

**Reply by GrampaWildWilly** [reply]
Paul has made a lot of progress with Vimeo content.  You need to get the latest beta version.  I explained how in a post upthread here.

I don't understand what you're looking for with "force download" or "not keep track of videos."  You'll have to elaborate that for me.  Maybe @paulrouget  understands.  I don't.

**Reply by Underdoggy66** [reply]
Downloading twice (or more) with the same resolution used to work with older releases (until January 2024 or so). Refreshing a web page brings back all the resolutions that are available, but it's not always possible to do a refresh when in the middle of a live stream.   Having an option in VDH to 'rescan' all the available resolutions would be a great thing to have..

**Reply by Mattmjs** [reply]
Open a duplicate tab of the tab you have open and it might allow downloading a second time.
